packages/models/src/errors.ts

```diff
@@ -126,7 +127,12 @@ export function makeApiProblemBuilder<T extends string>(errors: {
               },
             },
           },
-          (e) => e.response.data
+          (e) => {
+            const receivedProblem: Problem = e.response.data;
+            logger.warn(logMessage ?? "");
```

Comment:
The '""' would cause to log an empty line, we could avoid that logging at all if logMessage is not given

Reply:
> The '""' would cause to log an empty line, we could avoid that logging at all if logMessage is not given

You're right, fixed!